tasks/main.yml

```diff
@@ -1,27 +1,100 @@
-# SPDX-License-Identifier: MIT
 ---
-- name: Set platform/version specific variables
-  include_tasks: tasks/set_vars.yml
-
-# Examples of some tasks:
-- name: Ensure required packages are installed
+- name: "Ensure sudo is installed"
   package:
-    name: "{{ __template_packages }}"
-    state: present
-    use: "{{ (__template_is_ostree | d(false)) |
-             ternary('ansible.posix.rhel_rpm_ostree', omit) }}"
-
-- name: Ensure required services are enabled and started
-  service:
-    name: "{{ item }}"
-    state: started
-    enabled: true
-  loop: "{{ __template_services }}"
-
-- name: Generate /etc/{{ __template_foo_config }}
+    name: "sudo"
+    state: "present"
+  retries: 3
+  register: "sudo_installed"
+  until: "sudo_installed is succeeded"
+
+- name: "Set include directories variable"
+  set_fact:
+    sudo_include_dirs: "{{ sudo_sudoers_files
+    | selectattr('include_directories', 'defined')
+    | map(attribute='include_directories') | flatten }}"
+
+- name: "Set sudoer file paths variable"
+  set_fact:
+    sudo_file_paths: "{{ sudo_sudoers_files
+    | selectattr('path', 'defined')
+    | map(attribute='path') | flatten }}"
+
+- name: "Display all sudoers include directories"
+  debug:
```

Comment:
I do not remember exactly, but I think that the set_fact tasks above will output the result if you run playbook with `-v`, so these two debug tasks are not needed.